### ISSUE TITLE
anti-HLA A IgG3

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7104]
Preferred term label:
anti-HLA A IgG3

Definition (free text, please give PubMed ID)
Presence of anti-HLA A IgG3
29985463 [PMID] _23628391 [PMID] _33709433 [PMID] _ 28757117 [PMID]

Parent term (use hpo.jax.org/app)
NA

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Anti-HLA hyperimmunization ORPHA 2194

**Comment by pnrobinson** [comment]
added as HP:0034021
